### ISSUE TITLE
questions about MSVD

### ISSUE THREAD

**Issue by xixiareone** [niluthpol/multimodal_vtt#17]
The article mentions that "where they randomly chose 5 ground-truth sentences per video. We use the same setting when we compare with that approach".Does the training set, validation set and test set all take 5 sentences at random?
Not all sentences are used in training set, validation set and test set?

**Comment by niluthpol** [comment]
5 ground-truth sentences per video are used when compared with LJRV[24] (LJRV picked five ground truth descriptions per video). See Table 2 (Partition used by LJRV [24]).

**Comment by xixiareone** [comment]
can you share code about MSVD? thank you very much!




------------------&nbsp;原始邮件&nbsp;------------------
发件人:&nbsp;"Niluthpol Chowdhury Mithun"<notifications@github.com&gt;;
发送时间:&nbsp;2020年5月28日(星期四) 凌晨2:35
收件人:&nbsp;"niluthpol/multimodal_vtt"<multimodal_vtt@noreply.github.com&gt;;
抄送:&nbsp;"Gin"<xixiareone@foxmail.com&gt;;"Author"<author@noreply.github.com&gt;;
主题:&nbsp;Re: [niluthpol/multimodal_vtt] questions about MSVD (#17)





 
5 ground-truth sentences per video are used when compared with LJRV[24] (LJRV picked five ground truth descriptions per video). See Table 2 (Partition used by LJRV [24]).
